app/assets/javascripts/organization.coffee

```diff
@@ -0,0 +1,19 @@
+window.addAddressRow = (count) ->
+  newRow = $ """
+  <div class="form-group">
+    <label for="organization_addresses_attributes_""" + count + """_address">Mailing Address</label>
```

Comment:
CoffeeScript supports string interpolation, so this could be `<label for="organization_addresses_attributes_#{count}_address">Mailing Address</label>`, however I think this might be better as a placeholder text like the shipping tracking numbers, with perhaps an overall header for all the addresses.